coalib/bears/Bear.py

```diff
@@ -370,12 +370,21 @@ def check_prerequisites(cls):
         >>> SomeOtherBear.check_prerequisites()
         'really_bad_package is not installed. You can install it using ...'
 
+        >>> class anotherBear(Bear):
+        ...     REQUIREMENTS = {PipRequirement('bad_package','0.0.1')}
+
+        >>> anotherBear.check_prerequisites()
+        'bad_package 0.0.1 is not installed. You can install it using ...'
+
         :return: True if prerequisites are satisfied, else False or a string
                  that serves a more detailed description of what's missing.
         """
         for requirement in cls.REQUIREMENTS:
             if not requirement.is_installed():
-                return requirement.package + ' is not installed. You can ' + (
+                package = (
+                    requirement.package if not requirement.version
+                    else requirement.package + ' ' + requirement.version)
```

Comment:
I'am confused since this is a different repository. I'm not exactly sure how to push this. Should I clone the package_manager repo and make changes to it or ...... further clarification about how to do this would be helpful thanks.